test/TestUtilities/Test.Utility/CommandRunner.cs

```diff
@@ -95,9 +94,6 @@ public class CommandRunner
                         throw new TimeoutException($"{processName} timed out: {psi.Arguments}{Environment.NewLine}Output:{output}{Environment.NewLine}Error:{errors}");
                     }
                 }
-
```

Comment:
Curious if you saw some a behavior to decide to change this or just think it's wrong?

Reply:
It seems like not all of the output is captured and I suspect its because of this but I'm not 100% sure.  I don't think we need to be calling these methods since we're disposing of the process so I figured I'd remove it to see if it helps.